docs-ref-conceptual/authenticate-azure-cli-interactively.md

```diff
@@ -79,6 +79,13 @@ Here are some guidelines about the subscription selector to keep in mind:
 * The subscription selector is only available when using the `az login` command.
 * You aren't prompted to select a subscription when you're logging in with a service principal or managed identity.
 
+If you encounter any issue and want to rever to the previous login experience, run the following commands:
+
+```azurecli-interactive
+az config set core.login_experience_v2=off
```

Comment:
```suggestion
az account clear
az config set core.login_experience_v2=off
```

Reply:
@jiasli , @dcaro , the `az clear` isn't required, it just seems to make the shut-off of the feature cleaner in my thinking.  If there is a definite reason NOT to include it in the code block, please <resolve> my suggestion w/o adding the line.